src/app/map-stab/map-stab.component.ts

```diff
@@ -0,0 +1,102 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit, Output, ViewRef } from '@angular/core';
+import { map, mergeAll } from 'rxjs/operators';
+import * as coronaLocationsActions from '../../app/store/coronaLocations.action';
+import { selectMapEntitiesList } from '../store/coronaLocations.selector'
+import { selectCurrentItems } from '../store/coronaLocations.selector'
+import { MapService } from '../map.service'
+import { 
+  ActionType,
+  CesiumEvent,
+  CesiumService,
+  EventRegistrationInput,
+  MapEventsManagerService,
+  PickOptions } from 'angular-cesium';
+import { Observable } from 'rxjs';
+import { Store } from '@ngrx/store';
+import { IMapEntity } from '../mapEntity';
+
+import {MatDialog } from '@angular/material/dialog';
+import { DialogContentExampleDialogComponent } from "../dialog-content-example-dialog/dialog-content-example-dialog.component"
+@Component({
+  selector: 'app-map-stab',
+  templateUrl: './map-stab.component.html',
+  styleUrls: ['./map-stab.component.css']
+})
+export class MapStabComponent {
+  private _viewer = this._cesiumService.getViewer();
+  private _mapEntities: Observable<IMapEntity> = new Observable<IMapEntity>();
+  
+  private _counterForEntityId: number=0;
+  @Input()
+  markingFlag!:boolean;
+
+
+  constructor(private _eventManager: MapEventsManagerService,
+    public dialog: MatDialog,
+    private _mapService: MapService,
+    private _cesiumService: CesiumService,
+    private _store: Store<{storeCoronaLocation: any}>) { 
+  
+      const EVENT_REGISTRATION: EventRegistrationInput = {
+        event: CesiumEvent.LEFT_CLICK, 
+        priority: 0, 
+        pick: PickOptions.PICK_ONE 
+      };
+      let x= this._mapService.getAllMapEntities();
```

Comment:
you can move the loading and dispatch to the constructor of app.ts,
here you only need the select from store